RadixWallet/Core/FeaturePrelude/FactorSourceCard/FactorSourceCard+DataSource.swift

```diff
@@ -0,0 +1,77 @@
+// MARK: - FactorSourceCardDataSource
+struct FactorSourceCardDataSource {
+	let icon: ImageResource
+	let title: String
+	var subtitle: String?
+	var lastUsedOn: Timestamp?
+	var messages: [Message] = []
+	var accounts: [Account] = []
+	var personas: [Persona] = []
+
+	var hasAccountsOrPersonas: Bool {
```

Comment:
I would name this `hasEntities` or `hasAnyAccountsOrPersonas` 

(because `hasAccountsOrPersonas` sounds like something which would return an enum)

Reply:
`hasEntities` sounds good